_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
```

Comment:
going off of what Justin said, I think it could be valuable to add more information about how open source gets started at HackIllinois -- mentioning that hackers have the option to create their own projects like a traditional hackathon and make it open source vs. contributing (maybe doing this without using create/contribute or making it seem like two separate tracks since this is a problem we ran into in the past)

Reply:
I would definitely agree with what Api and Justin sare saying. I think it would valuable to add in more about how OS works at HackIllinois for those who aren't familiar. Then we also should include the fact that its not completely OS as we do want to make sure our public image includes both sides (even if it does favor OS)